src/Numerics/ODESolvers/GenericCallbacks.jl

```diff
@@ -1,137 +1,193 @@
 """
     GenericCallbacks
 
-A set of callback functions to be used with an `AbstractODESolver`
+This module defines interfaces and wrappers for callbacks to be used with an
+`AbstractODESolver`.
+
+A callback `cb` defines two methods:
+
+- `GenericCallbacks.init!(cb, solver, Q, param, t0)`, to be called at solver
+  initialization.
+
+- `GenericCallbacks.call!(cb, solver, Q, param, t)`, to be called after each time step:
+  the return value dictates what action should be taken:
+
+   * `0` or `nothing`: continue time stepping as usual
+   * `1`: stop time stepping after all callbacks have been executed
+   * `2`: stop time stepping immediately
```

Comment:
```suggestion
- `GenericCallbacks.call!(cb, solver, Q, param, t)`, to be called after each time step.
```

Reply:
We still do depend on the return value (at least for the time being).